dags/shredder.py

```diff
@@ -130,13 +131,10 @@
     ],
     # Needed to scale the highmem pool from 0 -> 1, because cluster autoscaling
     # works on pod resource requests, instead of usage
-    container_resources={
-        "request_memory": "13312Mi",
-        "request_cpu": None,
-        "limit_memory": "20480Mi",
-        "limit_cpu": None,
-        "limit_gpu": None,
-    },
+    container_resources=k8s.V1ResourceRequirements(
+        requests={"memory": "13312Mi"},
+        limits={"memory": "20480Mi"},
+    ),
```

Comment:
I suspect this was just copied from the probe scraper config since shredder doesn't use close to that much.  The [current pod](https://console.cloud.google.com/kubernetes/pod/us-west1/workloads-prod-v1/default/shredder-all-orjunh5d/details?inv=1&invt=AbjVJA&project=moz-fx-data-airflow-gke-prod) peaked at 5.3 GB.  I would set this lower since it's always running and the node has <24 GB allocatable and we don't want it to constantly scale unnecessarily
```suggestion
    container_resources=k8s.V1ResourceRequirements(
        requests={"memory": "7Gi"},
        limits={"memory": "9Gi"},
    ),
```